sdk/src/main/java/io/dapr/client/DaprClientGrpcAdapter.java

```diff
@@ -236,108 +248,53 @@
     return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
   }
 
-  /**
-   * Operation not supported for GRPC
-   * @throws UnsupportedOperationException every time is called.
-   */
   @Override
   public Mono<String> getActorState(String actorType, String actorId, String keyName) {
     return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
   }
 
-  /**
-   * Operation not supported for GRPC
-   * @throws UnsupportedOperationException every time is called.
-   */
   @Override
   public Mono<Void> saveActorStateTransactionally(String actorType, String actorId, String data) {
     return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
   }
 
-  /**
-   * Operation not supported for GRPC
-   * @throws UnsupportedOperationException every time is called.
-   */
   @Override
   public Mono<Void> registerActorReminder(String actorType, String actorId, String reminderName, String data) {
     return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
   }
 
-  /**
-   * Operation not supported for GRPC
-   * @throws UnsupportedOperationException every time is called.
-   */
   @Override
   public Mono<Void> unregisterActorReminder(String actorType, String actorId, String reminderName) {
     return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
   }
 
-  /**
-   * Operation not supported for GRPC
-   * @throws UnsupportedOperationException every time is called.
-   */
   @Override
   public Mono<Void> registerActorTimer(String actorType, String actorId, String timerName, String data) {
     return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
   }
 
-  /**
-   * Operation not supported for GRPC
-   * @throws UnsupportedOperationException every time is called.
-   */
   @Override
   public Mono<Void> unregisterActorTimer(String actorType, String actorId, String timerName) {
     return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
   }
 
-  /**
-   * Converts state options to map.
-   *
-   * TODO: Move this logic to StateOptions.
-   * @param options Instance to have is methods converted into map.
-   * @return Map for the state options.
-   * @throws IllegalAccessException Cannot extract params.
-   */
-  private Map<String, Object> transformStateOptionsToMap(StateOptions options)
-      throws IllegalAccessException {
-    Map<String, Object> mapOptions = null;
-    if (options != null) {
-      mapOptions = new HashMap<>();
-      for (Field field : options.getClass().getFields()) {
-        Object fieldValue = field.get(options);
-        if (fieldValue != null) {
-          mapOptions.put(field.getName(), fieldValue);
-        }
-      }
-    }
-    return mapOptions;
+  private <K> DaprProtos.InvokeServiceEnvelope getInvodeServceEnvelope(
```

Comment:
Typos + "get" gives the impression that it is just fetching an existing field. Please, rename to buildInvokeServiceEnvelope